objects/screen.c

```diff
@@ -112,14 +112,29 @@
  */
 
 /**
- * The screen number.
+ * The internal screen number.
  *
- * An integer greater than 1 and smaller than `screen.count()`. Please note that
- * the screen order isn't always mirroring the screen logical position.
+ * * The indeces are a continuous sequence from 1 to `screen.count()`.
+ * * It is **NOT** related to the actual screen position relative to each
+ *   other.
+ * * 1 is **NOT** necessarily the primary screen.
+ * * When screens are added and removed indexes **CAN** change.
```

Comment:
The plural from `index` is `indices`, I think. And I'd add a comma between `removed` and `indexes/indicies`.